django/contrib/auth/backends.py

```diff
@@ -8,8 +8,11 @@ class ModelBackend(object):
     """
     Authenticates against settings.AUTH_USER_MODEL.
     """
+    def __init__(self, request=None):
```

Comment:
It's not clear to me why these lines are needed or why `self.request = request` is assigned in `authenticate()`. At least some tests should be added for that if they are needed.